app-dropwizard/pom.xml

```diff
@@ -0,0 +1,39 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <parent>
+        <groupId>net.spals.appbuilder</groupId>
+        <artifactId>spals-appbuilder-parent</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>net.spals.appbuilder.plugins</groupId>
```

Comment:
Note that we use a slightly different `groupId` for plugins. This is to avoid confusion in the `bom` and elsewhere. Core appbuilder modules use `net.spals.appbuilder` whereas plugin modules use `net.spals.appbuilder.plugins`